docs/Recipes.md

```diff
@@ -142,6 +145,35 @@ export default makeConfig({
 });
 ```
 
+### With React Native 0.60 and newer
+
+React Native 0.60 removed Haste imports, so now in order to use out-of-tree platforms, you need to modify `InitializeCore` location and alias `react-native` to your out-of-tree library.
+
+Update your `haul.config.js` like so:
+```js
+import { makeConfig, withPolyfills } from "@haul-bundler/preset-0.60";
+
+export default makeConfig({
+  platforms: ['windows', 'ios', 'android'], // or ['windows'] if you only use windows platform
+  bundles: {
+    index: {
+      entry: entry: withPolyfills('./index.js', {
+        initializeCoreLocation: 'node_modules/react-native-windows/Libraries/Core/InitializeCore.js'
```

Comment:
wouldn't `require.resolve` be nicer?

Reply:
Well, it wouldn't work. `initializeCoreLocation` is joined with `root` option, which you can omit as in here and it will be set to `process.cwd()`. I'm not entirely sure why is that, but for sure doing `require.resolve` inside `withPolyfills` will not always work and we don't want to require users to specify `initializeCoreLocation` everytime with `require.resolve`.

Reply:
Get it. Paths are tricky :|